src/centos/scripts/start-marklogic.sh

```diff
@@ -12,6 +12,24 @@
 ###############################################################
 
 cd ~
+
+
+################################################################
+# Install Converters if required
+################################################################
+if [[ -z $INSTALL_CONVERTERS ]] ; then
```

Comment:
@jkerr5 One of the things I just noticed: 

If the `INSTALL_CONVERTERS ` value is set at all this will read as true. I'm assuming we want to have `INSTALL_CONVERTERS = false` to read as false.